Gem/Code/Include/DecalBus.h

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <AzCore/Math/Transform.h>
+#include <AzCore/EBus/EBus.h>
+#include <Atom/RPI.Public/Scene.h>
+
+namespace MultiplayerSample
+{
+    struct SpawnDecalConfig
+    {
+        AZ_RTTI(MultiplayerSample::SpawnDecalConfig, "{FC3DA616-174B-48FD-9BFB-BC277132FB47}");
+        inline static void Reflect(AZ::ReflectContext* context);
+
+        float m_scale = 1.0f;             // Scale in meters.
+        float m_opacity = 1.0f;           // How visible the decal is.
+        float m_attenutationAngle = 1.0f; // How much to attenuate based on the angle of the geometry vs the decal.
+        float m_lifeTime = 0.0f;          // Length of time the decal lives between fading in and out, in seconds.
+        float m_fadeInTime = 0.1f;        // Time it takes the decal to fade in, in seconds.
+        float m_fadeOutTime = 1.0f;       // Time it takes the decal to fade out, in seconds.
+        float m_thickness = 1.0f;         // How thick the decal should be on the z axis.
```

Comment:
Suggestion: Would be nice to add "Sec" at the end of these to make the unit more clear. This shows up later in the implementation, where the code looks like `float fadeInMsFloat = m_fadeInTime * 1000.0f`. The absence of the unit makes it a bit harder to see that this line is correct.